phpcs.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0"?>
 <ruleset name="CakePHP Core">
-    <rule ref="./vendor/cakephp/cakephp-codesniffer/CakePHP/ruleset.xml"/>
+    <rule ref="./vendor/cakephp/cakephp-codesniffer/CakePHP/ruleset.xml">
+        <exclude name="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint"/>
```

Comment:
This can be removed after mini-asset if updated to use param/return types and classes in this plugin which extend mini-asset's classes are updated.